sdk/core/Azure.Core/src/Response.cs

```diff
@@ -44,64 +41,85 @@ public override bool TryGetReasonPhrase(out string reasonPhrase)
         /// </summary>
         public virtual ResponseHeaders Headers => new ResponseHeaders(this);
 
-        internal HttpMessageSanitizer Sanitizer { get; set; } = HttpMessageSanitizer.Default;
-
-        internal RequestFailedDetailsParser? RequestFailedDetailsParser { get; set; }
+        // TODO(matell): The .NET Framework team plans to add BinaryData.Empty in dotnet/runtime#49670, and we can use it then.
+        private static readonly BinaryData s_EmptyBinaryData = new BinaryData(Array.Empty<byte>());
 
         /// <summary>
-        /// TBD.
+        /// Gets the contents of HTTP response, if it is available.
         /// </summary>
-        /// <param name="name"></param>
-        /// <returns></returns>
-        protected internal abstract bool ContainsHeader(string name);
+        /// <remarks>
+        /// Throws <see cref="InvalidOperationException"/> when <see cref="ContentStream"/> is not a <see cref="MemoryStream"/>.
+        /// </remarks>
+        public virtual BinaryData Content
```

Comment:
Can we defer this implementation?  This is suboptimal to have it here and not be able to delegate to the reusable component.